### ISSUE TITLE
Add option to specify media types to be generated on offerer in pub/sub/pubsub calls

### ISSUE THREAD

**Issue by prlanzarin** [mconf/bbb-webrtc-sfu#115]
Add the `medias` optional parameter which is an object/hash map of the following format:
```
medias: {
   audio: String|Array[String]|undefined ->  "sendrecv"|"recvonly"|"sendonly",
   video: String|Array[String]|undefined ->  "sendrecv"|"recvonly"|"sendonly",
   content: String|Array[String]|undefined ->  "sendrecv"|"recvonly"|"sendonly",
}
```
When the key values are arrays of the specified possible strings, it'll behave as a quantity specifier.

**Comment by prlanzarin** [comment]
This implicitly includes a full renegotiation support simulacrum.

**Comment by prlanzarin** [comment]
API was partially addressed in #131:
  - Single-media support for the `profiles` spec for audio, video and content.
  - Video and audio only support single renegotiation for now.
  - Content has full renegotiation support.

I'll close this one and open a new one with the remaining items.
